### ISSUE TITLE
Prompt user when file gets saved to the browser

### ISSUE THREAD

**Issue by Taguiar** [mozilla/thimble.mozilla.org#1909]
**What I propose to do**
This is the continuation of an UI update request by @humphd. I will be implementing a new feature that display icon that shows whenever a user is not logged into his thimble account and its document are saved locally to the browser.

**Steps to Reproduce**
User must be logged out
User makes changes and wait for 5 seconds
User jumps through files

**Expected behavior.**
I want the user to be prompt whenever a save occur.

**Actual behavior.**
Nothing is displayed when the document is changed of user moves between files.

**Instrument and Methodologies**
I will be using Firefox and google Chrome browsers to guarantee cross-browser.
I will be using Visual studio Code to ensure that my indentation is compatible with what the development team requires.

**Comment by Pomax** [comment]
Is this tied to a specific pre-existing issue? It feels like @humphd and @flukeout probably need to weigh in on this issue

**Comment by Taguiar** [comment]
Yes, it is tied to the one that I complete in brackets.

**Comment by Pomax** [comment]
@Taguiar can you mention which one a comment, so that github can setup an automatic crosslink?

**Comment by Taguiar** [comment]
Is the way that I did what you wanted?!

**Comment by Pomax** [comment]
looks like github does not place crosslinks for old comments... so:

see: https://github.com/mozilla/brackets/issues/607

**Comment by Pomax** [comment]
yes, thank you. Now there is a way for everyone to quickly get to the original issue, and the original (now closed) issue has a linkback to this one.
